packages/cna-template/template/_docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.7'
+
+services:
+  app:
+    build: .
+    image: <%= name %>/<%= name %>
+    container_name: <%= name %>
+    restart: always
+    volumes:
+      - /app/node_modules
```

Comment:
What's the purpose of permanent storage for `node_modules`?

Reply:
I added this to cache `node_modules` packages and speed up container execution.
Which I think is extra and the docker itself does that.